.github/workflows/master_release.yml

```diff
@@ -22,8 +22,8 @@ env:
   GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
   BASE_BUILD_NUMBER: 10000
     # Release version has to start with real version (23.2.0-....) for the docker image to build successfully.
-  RELEASE_VERSION: 23.3.0-A38
-  DEVELOPMENT_VERSION: 23.3.0-A39-SNAPSHOT
+  RELEASE_VERSION: 23.3.0
+  DEVELOPMENT_VERSION: 23.4.0-A1-SNAPSHOT
```

Comment:
I've heard somewhere that the dev version might be bumped to 25.1, would be good to double check with someone what should it be set to.

Reply:
Andrew said that the plan for now is to have 23.4.